test/validate/eia_test.py

```diff
@@ -3,39 +3,59 @@
 import logging
 
 import pytest
+import sqlalchemy as sa
 
 from pudl import validate as pv
-from test.conftest import skip_table_if_null_freq_table
+from pudl.helpers import get_parquet_table
 
 logger = logging.getLogger(__name__)
 
 
 @pytest.mark.parametrize(
-    "df_name,cols",
+    "table_name",
     [
-        ("bf_eia923", "all"),
-        ("bga_eia860", "all"),
-        ("boil_eia860", "all"),
-        ("frc_eia923", "all"),
-        ("gen_eia923", "all"),
-        ("gens_eia860", "all"),
-        ("gf_eia923", "all"),
-        ("own_eia860", "all"),
-        ("plants_eia860", "all"),
-        ("pu_eia860", "all"),
-        ("utils_eia860", "all"),
-        ("emissions_control_equipment_eia860", "all"),
-        ("boiler_emissions_control_equipment_assn_eia860", "all"),
-        ("denorm_emissions_control_equipment_eia860", "all"),
-        ("boiler_stack_flue_assn_eia860", "all"),
-        ("boiler_cooling_assn_eia860", "all"),
+        # Yearly EIA923 tables
+        "out_eia923__yearly_boiler_fuel",
+        "out_eia923__yearly_fuel_receipts_costs",
+        "out_eia923__yearly_generation",
+        "out_eia923__yearly_generation_fuel_combined",
+        # Monthly EIA923 tables
+        "out_eia923__monthly_boiler_fuel",
+        "out_eia923__monthly_fuel_receipts_costs",
+        "out_eia923__monthly_generation",
+        "out_eia923__monthly_generation_fuel_combined",
+        # Core EIA923 monthly tables
+        "core_eia923__monthly_boiler_fuel",
+        "core_eia923__monthly_fuel_receipts_costs",
+        "core_eia923__monthly_generation",
+        "core_eia923__monthly_generation_fuel_nuclear",
+        # Unaggregated EIA923 tables
+        "out_eia923__boiler_fuel",
+        "out_eia923__fuel_receipts_costs",
+        "out_eia923__generation",
+        "out_eia923__generation_fuel_combined",
+        # EIA860 tables
+        "core_eia860__assn_boiler_generator",
+        "out_eia__yearly_boilers",
+        "_out_eia__yearly_generators",
+        "out_eia860__yearly_ownership",
+        "out_eia__yearly_plants",
+        "_out_eia__plants_utilities",
+        "out_eia__yearly_utilities",
+        "core_eia860__scd_emissions_control_equipment",
+        "core_eia860__assn_yearly_boiler_emissions_control_equipment",
+        "out_eia860__yearly_emissions_control_equipment",
+        "core_eia860__assn_boiler_stack_flue",
+        "core_eia860__assn_boiler_cooling",
```

Comment:
Replace the PudlTabl method names with explicit table names. I'm pretty sure this list covers all the parameterized (yearly, monthly, unaggregated) tables we were testing before. But also, this is a stopgap until we get the no null cols test working in Dagster asset checks.